Assets/Scripts/Augment/BulletController.cs

```diff
@@ -14,7 +14,7 @@ public class BulletController : ProjectileController
     private float bulletSpeed = 50f;
 
     [SerializeField]
-    private int bulletsPerShot = 1;
+    public int bulletsPerShot { get; set; }
```

Comment:
Why did you turn it into a property? And please note that properties can't be serialized [according to the Unity docs](https://docs.unity3d.com/ScriptReference/SerializeField.html)

Reply:
changed it back, but public instead. Found no other way to retrieve bulletspershot info

Reply:
You could do a property with a private backing field (i.e., both of these kinda)

Reply:
Though I'm not sure if this is is something that is actually serialized?

Reply:
Fredrik showed me the `public fieldName => readFromField` method of reading private fields. Used this to retrieve the information.